src/root_console/components/AssignUsers.tsx

```diff
@@ -110,9 +115,11 @@ const AssignUsers = ({ users }) => {
 
   const availableUsers = users.filter(
     (user) =>
-      !currentUsers.some((currentUser) => currentUser.uuid === user.uuid),
+      !currentUsers.some((currentUser) => currentUser.uuid === user.uuid) && !user.is_root,
```

Comment:
Using !currentUsers.some(...) here is causing an error on my end, because !currentUsers translates to false if the array is empty or undefined. This results in false.some(...), which throws an error.

Reply:
I'm not seeing that error on my end. However, I think there's a bigger issue with the fact that getProjectUsers returns "user" objects with fewer properties than the getUsers route. I think projectUsers should have the same properties as regular users.

Reply:
Actually, I that error had to do that `currentUsers` itself was undefined because of the `data.users` call in a previous commit (as `data` has no `user` property). So that should be solved now. 

Regarding the `projectUser` properties, we could simplify this API call by just returning user uuids. We already have all user data in `users`, and could just filter the relevant user objects out based on a list of uuids returned by `getProjectUsers`.